command.ts

```diff
@@ -51,8 +51,17 @@ export interface BitfieldWithOverflowOpts extends BitfieldOpts {
 
 export type ClientCachingMode = "YES" | "NO";
 
```

Comment:
Please open another PR for this change. Other than that, it looks good to me. :+1:

Reply:
Sure, no problem. I created #214. After that is merged I can rebase this branch. Thanks!